### ISSUE TITLE
Embedding Projector doesn't allow to download T-SNE cluster data in a separate file for classification purposes

### ISSUE THREAD

**Issue by Sumit217** [tensorflow/tensorboard#1250]
Consider Stack Overflow for getting support using TensorBoard - they have a larger community with better searchability:

https://stackoverflow.com/questions/tagged/tensorboard

For bug reports, please include the following:

 - TensorBoard version (from pip package, also printed out when running `tensorboard`)
 - TensorFlow version if different from TensorBoard
 - OS Platform and version (e.g., Linux Ubuntu 16.04)
 - Python version (e.g. 2.7, 3.5)
 - For browser-related issues:
   - Browser type and version (e.g. Chrome 64.0.3282.140)
   - Screenshot if it's a visual issue
 
Please describe the bug as clearly as possible, and if possible provide a minimal example (code, data, and/or command line) to reproduce the issue.  Thanks!

**Comment by Sumit217** [comment]
@nfelt Would be grateful, if you can suggest me something on this?

**Comment by nfelt** [comment]
@Sumit217 I'm actually not very familiar with the embedding projector myself, and unfortunately we may not have time to address this feature request in the near future.

cc @dsmilkov @nsthorat @francoisluus in case anyone has suggestions for workarounds or has an idea for how this might be easily implemented.

**Comment by nsthorat** [comment]
Actually you already can. If you use the bookmarks bar in the bottom right, and save the bookmark, the computed t-SNE embeddings will be downloaded in JSON!

Let me know if you have trouble with that.

**Comment by Sumit217** [comment]
"  {"pca-0":0.2748771905899048,"pca-1":-0.37543976306915283,"pca-2":-0.2921373248100281,"pca-3":0.14536359906196594,"pca-4":-0.1794687956571579,"pca-5":-0.38032111525535583,"pca-6":0.14102117717266083,"pca-7":0.3352908492088318,"pca-8":0.41407617926597595,"pca-9":-0.1911829113960266,"tsne-0":11.142189254026334,"tsne-1":-8.650362966257541,"tsne-2":-9.99883919837444},{"pca-0":0.23078849911689758,"pca-1":0.4652230143547058,"pca-2":-0.1820177137851715,"pca-3":-0.33192309737205505,"pca-4":-0.2976536452770233,"pca-5":0.35050126910209656,"pca-6":0.053142569959163666,"pca-7":-0.0807952731847763,"pca-8":0.1354132741689682,"pca-9":-0.206165465465  "

@nfelt Thank you so much tagging these guys. 
@nsthorat Thank you so much for your response. I have tried that feature but i am not able to figure out, how do i make use of these to collect datapoints(sentences) hidden behind the clusters made, efficiently, in a file for labeling. If my embeddings make 5 clusters for my corpus, it would be great if i can get 5 different datasets(of sentences) seperately in a file. Can you suggest me something on these terms or may be something entirely different which can be used to solve this?

**Comment by nsthorat** [comment]
Hm... if you are looking for clusters for non-visual representation, you may want to just do something like K-means in python instead of using our tool. Our tool is very useful for visualization, but it sounds like you want to do something a little more custom and analytic.

sklearn has everything you'd want to do something like.

Am I missing some reason you want to use the embedding projector?

**Comment by Sumit217** [comment]
@nsthroat

Yes, you're right nikhil. That was one of the first option i explored but unfortunately not much luck with the output as my data has too much variance. 
You can assume, I am trying reverse engineering with tensorboard.

**Comment by vinayakumarr** [comment]
Is there any option to download the nearest neighbors. Also the slider for nearest neighbors is not working.

**Comment by vishalmhjn** [comment]
Why not have an option to download the space coordinates of the points after TSNE or PCA, it will help a lot apart from visualizing the data.
